### DISCUSSION TITLE
Should we add regex patterns for Email and Phone types?

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#84] [Suggestions] [answered]
Just putting this thought down before I forget. These are currently unvalidated free-text fields

**Comment by pjohnmeyer** [comment]
There's a json-schema format type for `email`: https://json-schema.org/understanding-json-schema/reference/string.html#email-addresses

Phone numbers could be regexed though. I'd suggest the [ITU E.123 international notation](https://en.wikipedia.org/wiki/E.123), e.g. +123 123 456 7890

**Reply by JSv4** [reply]
@pjohnmeyer, looks like this is still a loose end. Would you be open to revising the types as you proposed above?

**Reply by pjohnmeyer** [reply]
Sure, send it my way.

**Reply by JSv4** [reply]
@jacobyavis, actually, @pjohnmeyer has a bunch on his plate with the vesting changes. Do you mind taking this one? I can create an issue for it.

**Comment by JSv4** [comment, marked as answer]
Opened an issue for this. Assigned to @jacobyavis
